src/main/java/org/candlepin/resteasy/filter/StoreFactory.java

```diff
@@ -59,14 +60,29 @@ public class StoreFactory {
     @Inject
     public StoreFactory(Injector injector) {
         storeMap.put(Owner.class, new OwnerStore());
-        storeMap.put(Environment.class, new EnvironmentStore());
-        storeMap.put(Consumer.class, new ConsumerStore());
-        storeMap.put(Entitlement.class, new EntitlementStore());
-        storeMap.put(Pool.class, new PoolStore());
+        storeMap.put(Environment.class, new EnvironmentStore(
```

Comment:
If we've updated the classes below to use constructor injection, we could have the injector provide the store classes directly rather than still building them ourselves.

Reply:
I tried this and got "[Guice/InjectInnerClass]: Injecting into inner classes is not supported."

Reply:
gross.

Reply:
If we change these to static classes (as they probably should be), does that make Guice happier with the situation, or does it still get angry and refuse to help?

Reply:
It looks like that is it.